### DISCUSSION TITLE
Fetch all orders

### DISCUSSION THREAD

**Discussion by bakerhaider** [swellstores/community#493] [Help] [answered]
hello i want to fetch all orders from the backend api but my code only return the orders based on the logged in user and not all orders? 
![1ed1abfe9b4f49c717a732b9ca3588ed](https://github.com/swellstores/community/assets/33936834/44f52c3c-a8b8-4a0d-8f00-021fb7b54063)

**Comment by logeshswell** [comment, marked as answer]
Swell-js (Frontend API) fetches only the orders for the logged-in user, while swell-node (Backend API) returns all orders based on the given filters.

https://developers.swell.is/frontend-api/account-orders#list-account-orders
https://developers.swell.is/backend-api/orders/list-all-orders

**Reply by logeshswell** [reply]
Did you use swell-js instead of swell-node?
